Sources/System/Sockets/SocketDescriptor.swift

```diff
@@ -127,34 +153,113 @@ extension SocketDescriptor {
     @_alwaysEmitIntoClient
     public static var datagram: ConnectionType { ConnectionType(rawValue: _SOCK_DGRAM) }
 
-    /// Only available to the super user
+    /// Raw protocol interface. Only available to the super user
     ///
     /// The corresponding C constant is `SOCK_RAW`
     @_alwaysEmitIntoClient
     public static var raw: ConnectionType { ConnectionType(rawValue: _SOCK_RAW) }
+
+    /// Reliably delivered message.
+    ///
+    /// The corresponding C constant is `SOCK_RDM`
+    @_alwaysEmitIntoClient
+    public static var reliablyDeliveredMessage: ConnectionType {
+      ConnectionType(rawValue: _SOCK_RDM)
```

Comment:
I had omitted these, as they weren't really supported on Darwin, but I guess that's not System's call to make.